docs/user-guide/helm.md

```diff
@@ -1,87 +1,100 @@
 
-The Ambassador Edge Stack Helm chart is hosted in the
-[helm/charts](https://github.com/helm/charts) repository. Add this repo to your
-helm client with:
+The Ambassador Edge Stack Helm chart is hosted by Datawire and published at https://getambassador.io.
+Start by adding this repo to your helm client with:
 
 ```bash
-helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+helm repo add datawire https://www.getambassador.io
 ```
 
-**Note** that this will be changed soon as part of the Helm 2 deprecation.
-
 ## First Time Installation
 
 If you are installing the Ambassador Edge Stack for the first time on your host, complete the following directions:
 
-1. Install The Ambassador Edge Stack Chart with the following command:
+1. Create the `ambassador` namespace for Ambassador Edge Stack:
 
 ```
-helm install --namespace ambassador ambassador stable/ambassador
+kubectl create namespace ambassador
 ```
 
-This will install the necessary deployments, RBAC, Custom Resource Definitions, etc. for The Ambassador Edge Stack to route traffic. Details on how to configure Ambassador using the helm chart can be found in the Helm chart [README](https://github.com/helm/charts/tree/master/stable/ambassador).
-
-If you are using `Helm 2`, use the following command instead:
+2. If you are using Helm 3, install the Ambassador Edge Stack Chart with the following command:
 
 ```
-helm install --namespace ambassador --name ambassador stable/ambassador
+helm install ambassador --namespace ambassador datawire/ambassador
 ```
 
-2. Enable all Ambassador Edge Stack features with the following `kubectl` command:
+If you are using Helm 2, use the following command instead:
 
 ```
-kubectl apply -f https://www.getambassador.io/early-access/yaml/ambassador/ambassador-edge-stack-resources.yaml # FIXME: this URL is broken
+helm install --name ambassador --namespace ambassador datawire/ambassador
 ```
 
-The Helm chart deploys The Ambassador Edge Stack but does not create the Edge Stack resources necessary for authentication, rate limiting, automatic HTTPS, etc.
+This will install the necessary deployments, RBAC, Custom Resource Definitions, etc. for the Ambassador Edge Stack to route traffic. Details on how to configure Ambassador using the Helm chart can be found in the Helm chart [README](https://github.com/datawire/ambassador-chart/tree/master).
+
+**Note:** The Helm chart deploys the Ambassador Edge Stack, but it does not create the Edge Stack resources necessary for authentication, rate limiting, automatic HTTPS, etc.
 
-## Upgrade an Existing Installation
+## Upgrading an Existing Edge Stack Installation
 
-Upgrading your existing Ambassador Edge Stack is as simple as upgrading any other Helm release. Complete the following steps:
+**Note:** If your existing installation is not already running Ambassador **Edge Stack** as opposed to Ambassador Open Source, **do not use these instructions**. See "Migrating to Ambassador Edge Stack" below.
```

Comment:
If your existing installation is not already running the Ambassador **Edge Stack** (as opposed to the Ambassador API Gateway), **do not use these instructions**.